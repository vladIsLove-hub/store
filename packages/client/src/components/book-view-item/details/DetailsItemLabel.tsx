import React from 'react';

import { DetailsItemLabelProps } from './details.types';

const DetailsItemLabel: React.FC<DetailsItemLabelProps> = ({ text }) => {
  return (
    <div className="book-view__details-list-item-left">
      <span className="book-view__details-list-item-name">{text}</span>
    </div>
  );
};

export default DetailsItemLabel;
