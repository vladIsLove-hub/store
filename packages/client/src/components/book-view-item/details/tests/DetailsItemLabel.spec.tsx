import React from 'react';
import { render } from '@testing-library/react';

import DetailsItemLabel from '../DetailsItemLabel';

describe('DetailsItemLabel', () => {
  it('should render cuurently', () => {
    const { getByText } = render(<DetailsItemLabel text="test" />);
    expect(getByText('test')).toBeInTheDocument();
  });
  it('DetailsItemLabel snapshot', () => {
    const { getByText } = render(<DetailsItemLabel text="test" />);
    expect(getByText('test')).toMatchSnapshot();
  });
});
