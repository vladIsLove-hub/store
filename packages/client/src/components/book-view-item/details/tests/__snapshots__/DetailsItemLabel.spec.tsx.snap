// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailsItemLabel DetailsItemLabel snapshot 1`] = `
<span
  class="book-view__details-list-item-name"
>
  test
</span>
`;
